cmd/oras/internal/argument/file_windows.go

```diff
@@ -13,28 +13,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package main
+package argument
 
 import (
 	"strings"
 	"unicode"
 )
 
-// parseFileReference parses file reference on windows.
+// ParseFileReference parses file reference on windows.
 // Windows systems does not allow ':' in the file path except for drive letter.
```

Comment:
Windows does not allow `:`. Therefore, we should return error if the reference is invalid.

Reply:
Checked special reserved characters described in https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions